ramlfications/utils/parser.py

```diff
@@ -57,6 +59,42 @@ def resolve_inherited_scalar(item, inherit_from=[], **kwargs):
     return None
 
 
+def sort_uri_params(params, path):
+    if not params:
+        return params
+    # if this is hit, RAML shouldn't be valid anyways.
+    if isinstance(path, list):
+        path = path[0]
+
+    pattern = re.compile(r'\{(.*?)\}')
+    param_order = re.findall(pattern, path)
```

Comment:
[nit] I would have extracted and tested the pattern matching stuff appart from the uri stuff